Doc/library/sysconfig.rst

```diff
@@ -382,22 +382,24 @@ Other functions
 
    Examples of returned values:
 
+   Linux
+
    - linux-i586
+   - linux-i686
```

Comment:
linux-i586 and linux-i686 are both obsolete. linux-x86_64 would be a better example.